main.go

```diff
@@ -390,6 +393,90 @@ func updateGlobalBugCache(failedTestNames sets.String) {
 	}
 }
 
+func (a *Analyzer) identifyJobRunBugzillaFailer() {
+	for jrrKey, jrr := range a.RawData.JobRuns {
+		failingBugzillaComponents := sets.NewString()
+
+		for _, failedTest := range jrr.FailedTestNames {
+			// some tests are skipped
+			switch {
+			case strings.HasPrefix(failedTest, util.OperatorInstallPrefix) || strings.HasPrefix(failedTest, util.OperatorUpgradePrefix):
+				// these are handled by checking operator, no work on the string.
```

Comment:
there's no reason they can't work on the string (if you open a BZ and put the test name in the BZ, they will be found in the ci-search lookup).

whether that's useful or not i guess i'm less sure (you'll probably have a lot of periods of time when no bug exists associated w/ a particular "install operator foo", but you might at least want to try to do the bug lookup first.